src/pudl/ferc_to_sqlite/cli.py

```diff
@@ -39,6 +45,15 @@ def parse_command_line(argv):
         type=str,
         help="If specified, write logs to this file.",
     )
+    parser.add_argument(
```

Comment:
Should we add the `--clobber` flags back in to the `gcp_pudl_etl.sh` and `local_pudl_etl.sh` ? I think it doesn't matter for `gcp_pudl_etl` since it doesn't mount volumes, so it should always have a fresh output dir anyways.

Reply:
You're right that it doesn't matter because we aren't mounting volumes. I don't think we need to add it back.